config/images.yml

```diff
@@ -384,12 +384,16 @@
 # homepage
 "static/content/homepage/here-to-help.jpg":
   alt: "Back of an adviser's T-shirt saying 'here to help' at a teacher training event."
-
 "static/content/homepage/science-teacher.jpg":
   alt: "A chemistry teacher standing in a science classroom in front of a periodic table."
-
 "static/content/homepage/teacher-training-adviser.jpg":
   alt: "Teacher training advisers at an event."
+"static/content/homepage/directory1.jpg":
+  alt: "TODO"
+"static/content/homepage/directory2.jpg":
+  alt: "TODO"
+"static/content/homepage/directory3.jpg":
```

Comment:
This image has a bigger height than the other two @gazcarr @ethax-ross ?

![image](https://user-images.githubusercontent.com/87642394/226405163-6c185851-3753-4f3d-98bd-7084386ffb6d.png)